lib/astutils.cpp

```diff
@@ -238,6 +238,33 @@ bool isSameExpression(bool cpp, bool macro, const Token *tok1, const Token *tok2
     return commutativeEquals;
 }
 
+bool isContainerYield(const Token * const cond, Library::Container::Yield y, std::string name)
```

Comment:
name should be a constant reference.

Reply:
hmm.. I also somehow feel that `isContainerYield()` belongs in library.cpp instead.